packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -338,24 +399,70 @@ function WorklistDataGrid(props) {
     selectCell(cellRowIdColId, cellCoordinates);
   };
 
-  const handleRowSelection = (rowId, rowIndex, selectedCellCoordinates) => {
+  const handleRowSelection = (rowId, rowIndex, selectedCellCoordinates, enableSelectableRows, useLastSelectedRowAsAnchor) => {
```

Comment:
Who is passing these new arguments?  What does it mean to enable selectable rows?  Wouldn't the WorklistDataGrid be determining if the last selected row was used as an anchor since it is processing the key events?